src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -181,6 +183,7 @@ public static void main(final String[] args) {
     showMainFrame();
     new Thread(() -> setupLogging(GameMode.SWING_CLIENT)).start();
     HttpProxy.setupProxies();
+    checkLocalSystem();
```

Comment:
@DanVanAtta Do you have a suggestion for what acceptable network latency is from TripleA's perspective?  I was going to simply time how long the existing network check to GitHub takes and fail the check if it exceeds some threshold.